packages/lavamoat-node/test/index.spec.js

```diff
@@ -14,14 +15,26 @@ test('parseForPolicy - resolutions', async (t) => {
   }
 
   const policy1 = await parseForPolicy({ entryId, projectRoot })
+
+  // snapshot to prevent regression in policy sorting or output
+  t.snapshot(
+    readFileSync(path.join(projectRoot, 'lavamoat/node/policy.json'), 'utf8')
+  )
+
   t.deepEqual(policy1, {
     resources: {
+      'a>0zero': {
```

Comment:
any way to do `a,b,c` instead of `0zero,a,b`? would make far more sense at first glance

Reply:
I was going for a minimal diff. could be done, but IMHO not worth it.